pymc_marketing/mmm/base.py

```diff
@@ -16,11 +16,17 @@
 import pymc as pm
 import seaborn as sns
 from pymc.util import RandomState
+
+from scipy.optimize import Bounds, minimize
```

Comment:
The only part of the linter is failing is `isort` (see ci message):
![image](https://github.com/pymc-labs/pymc-marketing/assets/22996444/00056037-ba17-4d69-a4dd-f0e15da6b22d)
`from scipy.optimize import Bounds, minimize` should be below `from pytensor.tensor import TensorVariable` (I am curious why the pre-commit does not fail)